src/signals/settings/users/containers/Overview/components/DataView/components/DataViewHeader/components/DataViewHeaderRow/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { StyledTH, StyledTR } from 'signals/settings/users/containers/Overview/components/DataView/styled';
+
+const DataViewHeaderRow = ({ nodes, testId, spacer, spacerColSpan }) => (
+  <StyledTR data-testid={testId}>
+    {nodes.map((node, idx) => (
+      // eslint-disable-next-line react/no-array-index-key
+      <StyledTH key={idx} data-testid={`${testId}Heading`}>
+        {node}
+      </StyledTH>
+    ))}
+    {spacer && <StyledTH colSpan={spacerColSpan} />}
+  </StyledTR>
+);
+
+
+DataViewHeaderRow.defaultProps = {
+  testId: 'dataViewHeaderRow',
```

Comment:
This component doesn't have its own tests; why does it need a configurable `testId` prop?

Reply:
This `testId` is being used throughout the tests in `DataView.test`